discord/ext/test/factories.py

```diff
@@ -293,17 +293,18 @@ def dict_from_message(message: discord.Message):
     return out
 
 
-def make_attachment_dict(filename, size, url, proxy_url, id_num=-1, height=None, width=None):
+def make_attachment_dict(filename, size, url, proxy_url, id_num=-1, height=None, width=None, content_type=None):
     if id_num < 0:
         id_num = make_id()
     return {
         'id': id_num,
-        'filename': filename,
```

Comment:
Is there a reason why `filename` is after stuffs like `size, height, or width` ?

Seems odd to me...